### PR TITLE
refactor: type changes

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

N/A

### Description

- remove redundant `width`, `height` and `id` fields
- simplify types

### CODE REVIEW

**Review by doubleppereira — APPROVED** [review]

File: components/ModalTemplateAddGridWidget.vue [hunk 1]
```diff
@@ -38,12 +38,12 @@ const selectedProperties = computed(() => {
...
-    const newWidget: GridWidget = {
+    const newWidget: GridWidgetWithoutCords = {
       type: GRID_WIDGET_TYPE[selectedWidgetType.value.value as GridWidgetType],
-      width: 1,
-      height: 1,
+      w: 1,
+      h: 1,
       properties: { ...formValues.value },
-      id: uuidv4(),
+      i: uuidv4(),
```
Comment (JeneaVranceanu) [inline components/ModalTemplateAddGridWidget.vue:46]: To be honest `i` does not associate with `id` but rather with a counter from a loop 🙂
What is the idea driving these renames? 🤔

**Review by richtera — APPROVED** [review]

File: components/ModalTemplateAddGridWidget.vue (hunk 1 above)
Comment (dzbo) [reply components/ModalTemplateAddGridWidget.vue:46]: Thanks, actually I wanted to change that so `i` is not index of elements. Initially I thought that `i` suppose to be index but based on the library spec it's id of the element, so can be anything. That's why I removed previously added `id` field as redundant and changed that `i` is generated by `uuid` library.

Comment (richtera) [reply components/ModalTemplateAddGridWidget.vue]: We could call it "id" one additional character doesn't really matter much.

Comment (dzbo) [reply components/ModalTemplateAddGridWidget.vue]: Grid library already provides us `i` so if we add `id` like before we will have two identifiers which imo is not necessary.

**Pull request merged**